hopsworks-persistence/src/main/java/io/hops/hopsworks/persistence/entity/featurestore/storageconnector/s3/FeaturestoreS3Connector.java

```diff
@@ -67,6 +68,11 @@
   @Basic(optional = false)
   @Column(name = "name")
   private String name;
+  @Column(name = "server_encryption_algorithm")
+  @Convert(converter = FeaturestoreS3ConnectorEncryptionAlgorithmConverter.class)
```

Comment:
You don't need this and you don't need the converter. You just have to set the column type in the database to be `INT(11)` instead of `VARCHAR(100)`.

Javax persistence will take care of converting the enum to int and vice versa.